src/Microsoft.ML.Dnn/ImageClassificationTrainer.cs

```diff
@@ -499,7 +490,8 @@ public sealed class Options : TrainerInputBaseWithLabel
                 LabelColumnName = labelColumn,
                 ScoreColumnName = scoreColumn,
                 PredictedLabelColumnName = predictedLabelColumn,
-                ValidationSet = validationSet
+                ValidationSet = validationSet,
+                EarlyStoppingCriteria = new EarlyStopping()
```

Comment:
I think the default value of `EarlyStoppingCriteria` on any `Options` instance should be a `new EarlyStopping()` object. Can you move this to be always the default on `Options`?